crates/ironrdp-web/src/session.rs

```diff
@@ -977,6 +1013,13 @@ async fn connect(
         );
     }
 
+    if enable_audio {
+        debug!("Enabling audio with sample rate: {:?}", audio_sample_rate);
+        let audio_backend = WebAudioBackend::new(audio_sample_rate)
+            .map_err(|e| anyhow::Error::msg(format!("failed to initialize Web Audio backend: {e:?}")))?;
```

Comment:
suggestion: You can use the `context` method provided by the `anyhow::Context` trait.

```suggestion
            .context("failed to initialize Web Audio backend)?;
```